File: C#/Server/Server/GameInfo.cs
using System;
using System.Collections.Concurrent;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Server
{
    class GameInfo
    {

        public static int playerNumber = 1;

        public static ConcurrentDictionary<int, Boolean> gameStatus = new ConcurrentDictionary<int , Boolean>();

        public static String newPLayerID;

        public static ConcurrentDictionary<string , bool> players = new ConcurrentDictionary<string , bool>();



    }
}

File: Java/Server/src/GameInfo.java
import java.util.*;
import java.util.concurrent.ConcurrentHashMap;
import java.util.concurrent.ConcurrentLinkedQueue;

public class GameInfo {

    //Handles Global info about the game

    public static  int  playerNumber = 1;

    public static ConcurrentHashMap<Integer,Boolean> gameStatus = new ConcurrentHashMap<>();

    public static String newPLayerID;

    public static ConcurrentLinkedQueue<String> players = new ConcurrentLinkedQueue<>();
}
